src/chronometer.js

```diff
@@ -1,33 +1,47 @@
 class Chronometer {
   constructor() {
-    // ... your code goes here
+    this.currentTime = 0
+    this.intervalId = null
   }
 
   start(printTimeCallback) {
```

Comment:
Buenas tio!, no tiene sentido utilizar dos setInterval que ambos actualizan cada 1000 milisegundos.
La solucion sería así:
start(printTimeCallback) {
    // ... your code goes here
    this.intervalId = setInterval(() => {
      this.currentTime += 1;
      if (printTimeCallback) printTimeCallback();
    }, 1000);
  }